### PR TITLE
Update clear-access-levels.md

### PR DESCRIPTION
**Pull request by edgargarciamacrometa**
Added complete example

### CODE REVIEW

File: docs/sdks/clear-access-levels.md
Comment (jamesm-macrometa) [inline docs/sdks/clear-access-levels.md]:
```suggestion
// Email and password to authenticate client instance
```

File: docs/sdks/clear-access-levels.md
Comment (jamesm-macrometa) [inline docs/sdks/clear-access-levels.md]:
```suggestion
// Create an authenticated instance with an API key.
```

**Review by jamesm-macrometa — APPROVED** [review]
Minor comments, looks good!

**Comment by jamesm-macrometa** [comment]
@diana-macrometa Want me to go ahead and merge this one, or should I wait for your review?  The content is very similar to the last few of @edgargarciamacrometa's PRs

**Review by diana-macrometa — APPROVED** [review]
G2g once you address James' suggestions.

Btw, you only need one approval to merge.

**Pull request merged**
